### PR TITLE
Translate the CONTRIBUTING.md file into English

### PR DESCRIPTION
**Pull request by Jimmy385898**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,52 +1,57 @@
...
-## 贡献方式
+## Contribution mode
 
-您可以通过以下方式贡献：
+You can contribute by:
 
-1. **报告问题**：发现了文档中的错误或有改进的地方？请创建一个 Issue 来报告问题。
-2. **提出改进**：如果您有本项目的建议，请通过 Issue 提出您的想法。
+1. **Reporting problems**： Found an error or improvement in the documentation? Please create an Issue to report the problem.
+2. **Propose improvements**： If you have suggestions for this project, please send them through an Issue.
```
Comment (JackLau1222) [inline CONTRIBUTING.md:10]: `please send them through an Issue.` 
them? maybe `send us through an issue` be matched

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,52 +1,57 @@
...
 
-## 报告问题和提出建议
+## Report problems and make recommendations
 
-1. 在我们的 [Issues 页面](https://github.com/JackLau1222/OpenConverter/issues) 创建一个新的 Issue。
+1. In our [Issues](https://github.com/JackLau1222/OpenConverter/issues) to create a new Issue.
```
Comment (JackLau1222) [inline CONTRIBUTING.md:14]: Please create a new Issue in the [Issues](https://github.com/JackLau1222/OpenConverter/issues) page.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,52 +1,57 @@
...
 
-2. 选择相应的标签，例如“bug”或“enhancement”。
+2. Select the appropriate label, such as "bug" or "enhancement".
 
-3. 提供清晰而详细的说明，包括问题的复现步骤（如果是报告问题）或您的建议。
+3. Provide clear and detailed instructions, including steps to reproduce the problem (if reporting the problem) or your suggestions.
 
-## 提交项目
+## Submit project
```
Comment (JackLau1222) [inline CONTRIBUTING.md:20]: Submit Pull Request

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,52 +1,57 @@
...
 
-1. Fork 这个仓库并在本地克隆它的develop分支：
+1. Fork the repository and clone its develop branch locally:
```
Comment (JackLau1222) [inline CONTRIBUTING.md:22]: we develop this project base on main branch now

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,52 +1,57 @@
...
    
-4. 提交您的修改
+4. Commit your changes
     ```bash
     git add .
     git commit -m "Add/modify your-documentation"
     ```
     
-5. 推送到您的Fork
+5. Push to your Fork
```
Comment (JackLau1222) [inline CONTRIBUTING.md:38]: Push to your forked remote repo

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,52 +1,57 @@
...
     ```
     
-6. 创建一个 Pull Request（PR）：
+6. Create a Pull Request (PR) :
 
-- 到您的 Fork 页面，点击 "New Pull Request"。
-- 选择要将您的修改合并到的目标分支。
-- 提供清晰而简洁的 PR 描述。
-7. 我们将审查您的 PR，并在需要时提出建议或请求修改。
+- Go to your Fork page and click "New Pull Request".
```
Comment (JackLau1222) [inline CONTRIBUTING.md:45]: Go to your forked repository and click "New Pull Request".

**Review by JackLau1222 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (JackLau1222) [reply CONTRIBUTING.md:10]: don't forget to resove this conversation

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ Welcome to contribute to this repository! Your help is greatly appreciated and h
 
 You can contribute by:
 
-1. **Reporting problems**： Found an error or improvement in the documentation? Please create an Issue to report the problem.
+1. **Reporting problems**： Found an error or improvement in the documentation? Please send us through an issue.
 2. **Propose improvements**： If you have suggestions for this project, please send them through an Issue.
```
Comment (JackLau1222) [inline CONTRIBUTING.md:10]: maybe you modify worng code line

**Review by JackLau1222 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 7 above)
Comment (Jimmy385898) [reply CONTRIBUTING.md]: > maybe you modify wrong code line
I have resubmitted it, please review it

**Review by JackLau1222 — APPROVED** [review]
It looks good! Thanks for your contribution.

**Pull request merged**